.gitignore

```diff
@@ -3,7 +3,9 @@ xclaw
 *.mod
 *~
 .data
+.data_abl
```

Comment:
I don't think .data_abl or .output_abl are generated any longer with the new `Makefile_abl`.

Reply:
Good catch.